### PR TITLE
Update cisco-ise.py

### PR DESCRIPTION
**Pull request by ebuige**
Added two api's one ERS api (/ers/config/ancendpoint/clear) and one admin/monitoring api (/admin/API/mnt/Session/EndPointIPAddress/{ip}). The "clear" api removes a policy that has been assigned to an endpoint, the "session" api returns the session data from an active endpoint and writes this data to the context menu to be used as data enrichment or to access data from the session as needed. Because the admin/monitoring api's don't use the port (9060 for ERS api calls) a new url (without the port assigned) was created, a new http_request function was added to handle the admin calls and a new header to handle returned xml was added.

## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [x] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by orhovy** [comment]
Please fill the [contribution form](https://forms.gle/XDfxU4E61ZwEESSMA) so we can start the review.

File: Packs/cisco-ise/Integrations/cisco-ise/cisco-ise.py [hunk 1]
```diff
@@ -23,6 +24,12 @@
     'Accept': 'application/json',
     'Connection': 'keep_alive'
 }
+
+DEFAULT_HEADERS1 = {
```
Comment (yaakovi) [inline Packs/cisco-ise/Integrations/cisco-ise/cisco-ise.py:28]: should be named: `DEFAULT_ADMIN_API_HEADERS`

File: Packs/cisco-ise/Integrations/cisco-ise/cisco-ise.py [hunk 2]
```diff
@@ -63,6 +70,43 @@ def http_request(method, url_suffix, params={}, data=None, headers=DEFAULT_HEADE
...
+        err_msg = f'Error in API call to Cisco ISE Integration [{response.status_code}] - {response.reason}, {message}'
+
+        return_error(err_msg)
+
+    if response.status_code in (201, 204):
+        return
+    try:
+        response = response.content
+    except ValueError:
+        demisto.results("this is the error ")
```
Comment (yaakovi) [inline Packs/cisco-ise/Integrations/cisco-ise/cisco-ise.py:105]: remove this line

**Review by yaakovi — COMMENTED** [review]
looks good.
please bump the pack version and update the release notes.
you can do this using the `demisto-sdk`.
see: https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/update_release_notes/README.md

**Comment by yaakovi** [comment]
@ebuige since you added 2 new commands, you should also update the yml file of the integration.

**Review by yaakovi — APPROVED** [review]

**Pull request merged**